client/ayon_houdini/plugins/publish/validate_review_colorspace.py

```diff
@@ -57,6 +57,18 @@ def apply_settings(cls, project_settings):
 
     def process(self, instance):
 
+        rop_node = hou.node(instance.data["instance_node"])
+
+        # This plugin is triggered when marking render as reviewable.
+        # Therefore, this plugin will run on over wrong instances.
+        # TODO: Don't run this plugin on wrong instances.
+        # This plugin should run only on review product type
+        # with instance node of opengl type.
+        if rop_node.type().name() not in {"opengl", "flipbook"}:
+            self.log.debug("Skipping Validation. Rop node {} "
+                           "is not an `opengl` or `flipbook` node.".format(rop_node.path()))
+            return
```

Comment:
Why is this needed? This is not the case anymore right? Now it triggers on `rop.opengl` family?

Reply:
https://github.com/ynput/ayon-houdini/pull/167/commits/39637766f65e6723d2a1179b4906ab0ce908f424